index.html

```diff
@@ -0,0 +1,158 @@
+      rel="stylesheet"
+      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="style2.css" />
+    <title>Chester</title>
+  </head>
+  <body>
+    <header>
+      <!-- mobile  -->
+      <div class="header-sect">
+        <span class="clas"></span>
+        <div id=mob-nav>
+          <ul class="menu-1">
+            
+             
+              <li><a href="#">Program</a></li>
+              <li> <a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li><li>
+           <a href="about.html">About</a></li>
+          </ul>
+        </div>
+        <span class="clas"></span>
+      </div>
+      <!-- Desktop  -->
+      <nav>
+        <div class="media">
+          <ul>
+            <li>
+              <a href="https://web.facebook.com/rotshidzwa.mavhungu.5/"><i class="fab fa-facebook-f"></i></a>
+            </li>
+            <li>
+              <a href="https://twitter.com/footballlz"><i class="fab fa-twitter"></i></a>
+            </li>
+            <li><a>English</a></li>
+            <li><a>My page</a></li>
+            <li><a>Logout</a></li>
+          </ul>
+        </div>
+        <div class="navba">
+          <a href="index.html"><img src="images/Capture.png" class="navImage" alt="" /></a>
+          <ul>
+            <li><a href="#">program</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li><a href="./about.html">about</a></li>
+            <li><button type="button">CT Campaign</button></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <main> 
+      <section class="ban">
+        <h1>The 2022 
+            <br>sports
+             <br>celebration</h1>
+        <div class="ban-summary">
+          <p>
+            FIFA is a non-profit organization that describes itself as an international governing body of association football, futsal and beach soccer. It is the highest governing body of association football.
+          </p>
+        </div>
+        <h2>2021.10.12(tue) ~ 15(fri)</h2>
+        <p class="place">@ stellanbosch, South africa</p>
+      </section>
+
+      <section class="prog">
+        <h3>Main Program</h3>
+        <hr class="line" />
+
+        <div class="prog-line">
+          <div id="prog-0" class="cards">
+            <img src="images/calendar.svg" alt="" />
+            <p class="title">Calendar</p>
+            <p>date and time are important  to everyone around the world</p>
+          </div>
```

Comment:
- [ ] Kindly make the `cards` wrapper an article to make your code more semantic and readable.
- [ ] Also kindly make the `title` a heading.
- [ ] How about aligning the description to the center so that your design matches the original design 🤷🏻?
```
<article id="prog-0" class="cards">
    <img src="images/calendar.svg" alt="" />
    <h4 class="title">Calendar</h4>
    <p>date and time are important  to everyone around the world</p
</article>
```

- [ ] In the mobile design, the items are not well centered as seen in the original design. Kindly correct the alignment.

| Current Design | Original Design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/4247740/164853429-713161ac-5a75-4ac2-8c03-3389ea7f1cfe.png) | ![image](https://user-images.githubusercontent.com/4247740/164854068-19b6b987-e1fb-4e97-ad42-caf9f76941d2.png) |